pb_app/lib/submission_form.dart

```diff
@@ -104,24 +110,71 @@ class _SubmissionFormScreenState extends State<SubmissionFormScreen> {
   }
 }
 
-class _Card extends StatelessWidget {
-  const _Card(this.title);
-
+class _Card extends StatefulWidget {
+  const _Card(
+    this.title,
+  );
```

Comment:
```suggestion
  const _Card(this.title);
```

revert pure style change